packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -20,24 +20,47 @@ The `core`, `support`, `layout` npm packages have been deprecated and replaced w
+The `dist` folder structure has been updated for the `@cmsgov/design-system` npm package. We recommend only importing files from the `dist` directory moving forward.
 
-For JavaScript imports, this will likely result in no changes. While JavaScript files were always available in the `src` directory before, it was never recommended to import the source version directly, so your project likely will not contain any `src` folder imports for React components.
+```
+└── @cmsgov/design-system/dist
+    ├── components/
+    │   └── index.js      Compiled JS (CommmonJS)
+    ├── css/
+    │   └── index.css     Compiled CSS
+    ├── esnext/
+    │   └── index.esm.js  Compiled JS (ES Module)
+    ├── fonts/
+    ├── images/
+    └── scss/             Uncompiled CSS
+```
+
+### Importing SCSS/CSS
+
+If you were importing Sass directly from our packages’ `src` directories, we recommended updating your import paths to import from `dist`.
 
-For your Sass imports, we recommend everyone to update your import paths. We previously did not distribute the Sass files in our packages’ `dist` directories, so you likely had an import path that looked like this:
+We previously did not distribute the Sass files in our packages’ `dist` directories, so you likely had an import path that looked like this:
 
 ```
-@import "~@cmsgov/design-system-core/src/index.scss";
+@import "~@cmsgov/design-system-core/src/index";
 ```
 
-In v2, it will be changed to
+In CMSDS v2, it will be changed to
 
 ```
-@import "~@cmsgov/design-system/dist/index.scss";
+@import "~@cmsgov/design-system/dist/scss/index";
 ```
 
+Imports to `@cmsgov/design-system-layout` or `@cmsgov/design-system-support` are now included in the main `scss` entry point. If you would like to only import specific styles, see the `@cmsgov/design-system` [`README.md`](https://github.com/CMSgov/design-system/blob/master/packages/design-system/README.md#file-structure) for more info on our SASS architecture.
+
+### Importing Javascript
+
+V2 adds support for an ES module version of our JS, which can be found in the `dist/esnext` directory. Our `package.json` has been updated to point to the new ES module entry point when possible, so for most users this won't require any change. See the documentation on [importing React components]({{root}}/startup/components/#named-imports) for more information on importing Javascript and ES module support.
+
+Your project also should not contain any `src` folder imports for React components. While JavaScript files were always available in the `src` directory before, it was never recommended to import the source version directly.
+
 ## Fonts and Images
 
 Fonts and images are now stored in `@cmsgov/design-system/dist/fonts` and `@cmsgov/design-system/dist/images`. Previously they were stored in `@cmsgov/design-system/fonts` and `@cmsgov/design-system/images`.
```

Comment:
I think it would be nice to put these code blocks into a multiline block for easier reading like this

Fonts and images are now stored in:

```
@cmsgov/design-system/dist/fonts
@cmsgov/design-system/dist/images
```

Previously they were stored in:
```
@cmsgov/design-system/fonts
@cmsgov/design-system/images
```

Reply:
Theres already an example code block of that a few lines down